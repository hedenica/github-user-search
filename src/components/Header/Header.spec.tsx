import { render } from '@testing-library/react';

import { Header } from '.';

describe('Header', () => {
  it('renders correctly', () => {
    const { container } = render(<Header />);

    expect(container).toMatchSnapshot();
  });
});
