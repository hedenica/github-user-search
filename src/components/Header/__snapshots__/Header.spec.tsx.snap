// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div>
  <div>
    <img
      alt="github-icon"
      src="[object Object]"
    />
    <h2>
      github-search
    </h2>
  </div>
</div>
`;
